src/common/footer/ExtendedFooter.jsx

```diff
@@ -1,14 +1,13 @@
-import { useContext } from 'react';
 import { Link } from 'react-router-dom';
 import { IoAddSharp, IoLogoRss } from 'react-icons/io5';
 import { MdManageSearch } from 'react-icons/md';
 import { FaDiscord } from 'react-icons/fa';
 import { BsTwitter, BsGithub } from 'react-icons/bs';
 import LogoLight from 'images/img-logo-light.svg';
-import { SearchContext } from 'common/search/search-context';
+import { useSearchContext } from 'common/search/search-context';
 
 const ExtendedFooter = () => {
-  const { showShareModal, setShowShareModal } = useContext(SearchContext);
+  const { showShareModal, setShowShareModal } = useSearchContext();
```

Comment:
Why don't we need to pass search context here?

Reply:
I have replaced `useContext` with a custom hook `useSearchContext`
It creates an abstraction, but in turn uses `SearchContext`

Reply:
I checked that now!

A couple of questions: 

- Can you conform all the errors in common are resolved?
- How have you tested such that the functionalities are not broken?

Reply:
Hi @atapas 
I tested out the app and it seems to be working fine with all my changes.

However, we do get the pop-up to fix the other lint issues (from `play`) folder. This PR contains lint fixes for files within the `common` folder.